actors/builtin/shared.go

```diff
@@ -43,6 +43,10 @@ type MinerAddrs struct {
 	Worker addr.Address
 }
 
+// This type is required to construct a miner (in power_actor) and change a miners multiaddress (in miner_actor)
+// the type is defined here to work around a circular dependency between the actors.
+type MultiAddress = []byte
```

Comment:
I think we'd be ok just using `[]byte` in place, since actor code never inspects or cares about the internals.

Reply:
I made it a separate type as a means of providing documentation, i'm a bit surprised by this comment coming from you :smile: